iptables/iptables.go

```diff
@@ -72,11 +72,12 @@ func (e ChainError) Error() string {
 }
 
 func probe() {
-	if out, err := exec.Command("modprobe", "-va", "nf_nat").CombinedOutput(); err != nil {
-		logrus.Warnf("Running modprobe nf_nat failed with message: `%s`, error: %v", strings.TrimSpace(string(out)), err)
+	path, err := exec.LookPath("iptables")
+	if err != nil {
+		return
```

Comment:
Should this still print a warning perhaps, if `iptables` cannot be found?

Reply:
Sure, why not.

Note: detectIptables, where I copied that chunk from, doesn't either.

Reply:
Ah, hm, yes, if it wasn't there (not sure why though 🤔 😅) (I'm not a maintainer in this repository, so perhaps there was a reason for that 😅)